### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by hntrhfr**
i updated the github API token guide (and also fixed some auto formatting stuff)

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -4,14 +4,18 @@
...
 
-- Create an access token for the GitLab API. This will be required to install our private NPM packages. The token has to have API scope.  
-Link to the GitLab settings: [Manage GitLab Access Tokens](https://gitlab.mediacube.at/-/profile/personal_access_tokens)
+- Create an access token for the Github API. This will be required to install our private NPM packages. You can follow [this](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) guide.
+
+  TLDR:
+
+  - Go to settings -> Developer settings -> Personal access tokens -> Tokens (classic)
+  - create a token with at least the scope `read:packages`
+  - `npm login --scope=@OWNER --auth-type=legacy --registry=https://npm.pkg.github.com` with your github user and the token as password
```
Comment (lucapircher) [inline Readme.md]:
```suggestion
  - `npm login --scope=@a11yphant --auth-type=legacy --registry=https://npm.pkg.github.com` with your github user and the token as password
```

We could insert the correct scope in the command here. I can already see myself trying to login for the `@OWNER` scope...

**Review by lucapircher — CHANGES_REQUESTED** [review]
Please also update the title in the readme file. We should have done that a long time ago

**Comment by hntrhfr** [comment]
> Please also update the title in the readme file. We should have done that a long time ago

done

**Review by hntrhfr — COMMENTED** [review]
nice

**Review by lucapircher — APPROVED** [review]

File: Readme.md [hunk 2]
```diff
@@ -1,17 +1,19 @@
-# A11y Challenges
-
-![coverage](https://gitlab.mediacube.at/a11yphant/a11yphant/badges/develop/coverage.svg)
```
Comment (thomasdax98) [inline Readme.md:3]: Why did you remove the coverage? Doesn't that work on Github?

Comment (lucapircher) [reply Readme.md:3]: Github has no integrated support for code coverage. We will have to setup an external tool like codecov or codeclimate to bring such functionality back.

**Pull request merged**